README.md

```diff
@@ -21,7 +28,7 @@ out here!)_
   1. [Introduction](#1-introduction)
   2. [Dependencies](#2-dependencies)
   3. [Building-From-Source](#3-building-from-source)
-  4. [Running Models](#4-running-models)
+  5. [Running Models](#5-running-models)
```

Comment:
Looks like we have to adjust the listing here.

Reply:
Sure, I'll make the changes. Thanks a lot.